### ISSUE TITLE
Fine tune opening dashboard view and remote session instructions

### ISSUE THREAD

**Issue by brimoor** [voxel51/fiftyone#62]
Here's what the no dataset page of the FiftyOne Dashboard currently looks like:

<img width="881" alt="Screen Shot 2020-05-13 at 1 12 54 PM" src="https://user-images.githubusercontent.com/25985824/81843360-7ce10a00-951b-11ea-8a37-78148f0486d1.png">

If the user has a dashboard open, then they've already figured out how to run `fo.launch_dashboard()`, right? So, perhaps we should extend the help a bit to show the full workflow:

```py
import fiftyone as fo

# Load your FiftyOne dataset
dataset = ...

# Launch your dashboard locally
# (if you're reading this from your dashboard, you've already done this!)
session = fo.launch_dashboard()

# Load a dataset 
session.dataset = dataset

# Load a specific view into your dataset
session.view = view
```

Remote connections are a bit of a special case, so perhaps they should be hidden in a view that appears only after clicking on a `Remote session?` link or something similar. Then perhaps that help page should show bifurcated local/remote instructions like this: 

## On your remote machine

```py
import fiftyone as fo

# Load your FiftyOne dataset
dataset = ...

# Launch the dashboard that you'll connect to from your local machine
session = fo.launch_dashboard(remote=True)

# Load a dataset 
session.dataset = dataset

# Load a specific view into your dataset
session.view = view
```

## On your local machine

```bash
# Configure port forwarding to access the session on your remote machine
ssh -L 5151:127.0.0.1:5151 username@remote_machine_ip

# Launch the dashboard
# NOTE: I'm going to support this via the CLI as a shortcut for launching a
# dashboard that you intend to connect to remotely
fiftyone remote   
```
